mmcv/transform/processing.py

```diff
@@ -0,0 +1,382 @@
+import random
+import warnings
+from typing import Dict, List, Optional, Sequence, Tuple, Union
+
+import numpy as np
+
+import mmcv
+from mmcv.utils import build_from_cfg
+from .base import BaseTransform
+from .builder import TRANSFORMS
+from .wrappers import Compose
+
+Number = Union[int, float]
+
+
+@TRANSFORMS.register_module()
+class CenterCrop(BaseTransform):
+    """Center crop the image. If the crop area exceeds the original image and
+    ``pad_mode`` is not None, then pad the original image first.
+
+    Args:
+        crop_size (Union[int, Tuple[int, int]]):  Expected size after cropping
+            with the format of (h, w).
+        pad_val (Union[Number, Dict[str, Number]], optional): A dict for
+            padding value. Defaults to dict(img=0, seg=255).
```

Comment:
might provide more explanation:
- if the input is a number, what is the pad behavior
- what are function of keys and values in the dict
- might be the mode is constant, the pad_val will work

Reply:
This is specified in `Pad`. I will add a explanation to tell users so.